commands/create_kubeconfig_test.go

```diff
@@ -10,21 +10,41 @@ import (
-        "create_date": "2017-04-18T20:17:14.544037411Z",
-        "description": "test",
-        "id": "48:b9:01:ce:34:8f:b2:08:d3:4f:8c:bb:5e:2f:d7:b6:bc:ae:5c:98",
-        "ttl_hours": 24
-      }`))
-
+		if r.Method == "GET" && r.URL.String() == "/v4/clusters/test-cluster-id/" {
+			// return cluster details
+			w.WriteHeader(http.StatusOK)
+			w.Write([]byte(`{
+				"id": "test-cluster-id",
+				"name": "Name of the cluster",
+				"api_endpoint": "https://api.foo.bar",
+				"create_date": "2017-11-20T12:00:00.000000Z",
+				"owner": "acmeorg",
+				"kubernetes_version": "",
+        "release_version": "0.3.0",
+				"workers": [
+					{"aws": {"instance_type": "m3.large"}, "memory": {"size_gb": 5}, "storage": {"size_gb": 50}, "cpu": {"cores": 2}, "labels": {"foo": "bar"}},
+					{"aws": {"instance_type": "m3.large"}, "memory": {"size_gb": 5}, "storage": {"size_gb": 50}, "cpu": {"cores": 2}, "labels": {"foo": "bar"}},
+					{"aws": {"instance_type": "m3.large"}, "memory": {"size_gb": 5}, "storage": {"size_gb": 50}, "cpu": {"cores": 2}, "labels": {"foo": "bar"}}
+				]
+			}`))
+		} else if r.Method == "POST" && r.URL.String() == "/v4/clusters/test-cluster-id/key-pairs/" {
+			// return new key pair
+			w.WriteHeader(http.StatusOK)
+			w.Write([]byte(`{
```

Comment:
No blocker, but using a struct and marshalling it to string will probably make your life a bit easier with this kind of thing

Reply:
I like the fact that I can see the JSON verbatim. As long as I only have a few of these, I think we're fine.